tests/unit/tuf_repository_service_worker/test_repository.py

```diff
@@ -31,6 +31,32 @@ def test_refresh_settings(self):
         )
         assert isinstance(test_repo._settings, repository.Dynaconf) is True
 
+    def test_refresh_settings_with_worker_settings_arg(self):
+        FAKE_SETTINGS_FILE = "/data/mysettings.ini"
+        fake_worker_settings = Dynaconf(
+            settings_files=[FAKE_SETTINGS_FILE],
+            envvar_prefix="RSTUF",
+        )
```

Comment:
Nit: not sure if we should call it a `file` or a `path` as it's a string.
```suggestion
        FAKE_SETTINGS_PATH= "/data/mysettings.ini"
        fake_worker_settings = Dynaconf(
            settings_files=[FAKE_SETTINGS_PATH],
            envvar_prefix="RSTUF",
        )
```

Reply:
I am okay leaving it that way, it's just strange we call them `settings_files` as they are not actually loaded files.
Whatever you decide @KAUTH.

Reply:
Indeed on the one hand it follows the [Dynaconf implementation](https://github.com/dynaconf/dynaconf/blob/18195722db7d895636053f2d48598f905710b0ab/dynaconf/base.py#L46) and on the other it's a string of the file path.
I renamed it `FAKE_SETTINGS_FILE_PATH`, but in any case since it's given as an arguement to an external keyword we cannot change that and I believe it's not going to cause too much confusion.